tests/ga/test_extension.py

```diff
@@ -567,6 +573,299 @@ def test_ext_handler_reporting_status_file(self, *args):
 
         self.assertEquals(expected_status_json, actual_status_json)
 
+    def test_handler_configuration_complete_configuration(self, *args):
```

Comment:
need more tests for invalid manifests

Reply:
Added more tests, and got the coverage of the new code to close to 100% (pretty printing of the objects not tested, and sorting in other ways not included)